src/SFA.DAS.ApplyService.Web/Controllers/Roatp/RoatpApplicationController.cs

```diff
@@ -195,8 +195,7 @@ private static Application.Apply.Start.StartApplicationRequest BuildStartApplica
                         //RequestedFeedbackAnswered = false
                     }).OrderBy(section => section.SectionNo).ToList(),
                     //Status = "Draft",
-                    //IsActive = sequence.IsActive,
-                    //NotRequired = sequence.NotRequired,
+                    //IsActive = sequence.IsActive,                    
```

Comment:
Do you need to include saving the config for the specifying the task list sections here?